pkg/utils/io.go

```diff
@@ -48,3 +48,25 @@ func LogClose(fh io.Closer) {
 	}
 	zap.S().Errorf("Failed to close file: %s", err)
 }
+
+// IsValidSetupDirectory checks if directory is valid for setup configuration
+func IsValidSetupDirectory(name string) (bool, error) {
+	_, err := os.Stat(name)
+	if err != nil {
```

Comment:
This can return an error when you don't have permissions to install to that directory.

Reply:
yes, that's why I didn't name it `IsEmptyDirectory` from the beginning but maybe the comment was misleading :)

Reply:
I mean check the error and don't return true when you don't have permissions (or there is some other error).